### DISCUSSION TITLE
Give us a way to see at bird's eye view if a log has errors in it

### DISCUSSION THREAD

**Discussion by vesper8** [opcodesio/log-viewer#101] [Ideas]
One of the things I miss from arcanedev/log-viewer is the ability to see a summary of what a log file contains without having to first click on it. In my projects, I often have a lot of debug/notice logging active which is never critical stuff, but if there are any error or critical logs then these need my immediate attention. Currently when looking at the landing page of your log viewer, you might see a list of log files but there's no way to know if any of them have errors in them without clicking over each of them.

There's a few ways you could add a visual cue to the list of logs on the left column to indicate if some logs need more attention than others. There is certainly limited space to work with here.. I'm thinking perhaps the way I would prefer is if the background colour of the logs in the left list would be represented by the severity of the logs it contains. So if a log contains errors, it would be a red gradient, and the gradient could be more or less red depending on the number of errors it contains?

Personally I'd be happy if this was only affected by errors / critical / emergency log levels. And I understand some people may not like adding colour to this dashboard so this could be enabled/disabled via the config file or the UI maybe?

Just some food for thought.

Cheers!

**Comment by MGeurts** [comment]
When I install this application, it does not show the summary header, as stated in the documentation. Why not? It's so useful.

![Summary](https://github.com/opcodesio/log-viewer/assets/5404867/4f2b9861-c596-4255-859f-9a4c3eb274af)

**Reply by arukompas** [reply]
Hey @MGeurts , the design has changed a little bit and these buttons have been moved into a dropdown to make room for a search bar and other filter options in the future. 

But I do agree that an overview of some kind would be very useful here!

**Reply by MGeurts** [reply]
OK. I see.
It's important for us to see the most important statistics at first glance without using any controls.
At least the **warning** and **error** statistics should be visible at first glance and 1 click on them should lead to the details.

As a second remark, the current filter, hidden in the dropdown, shows the log catagories per log file.
To monitor a website, it would be nive to have statistics over **all log files**.
Thanks.
